server-base/src/main/java/org/apache/kylin/rest/service/HybridService.java

```diff
@@ -43,8 +44,7 @@
     @Autowired
     private AclEvaluate aclEvaluate;
 
-    public HybridInstance createHybridCube(String hybridName, String projectName, String modelName,
-            String[] cubeNames) {
+    public HybridRespone createHybridCube(String hybridName, String projectName, String modelName, String[] cubeNames) {
```

Comment:
"HybridRespone" is from the rest layer, it shouldn't be referenced at the service layer. Here the interface should be "HybridInstance"